### ISSUE TITLE
CMake: installation fails if DESTDIR variable is specified

### ISSUE THREAD

**Issue by tpimh** [libgit2/libgit2#5423]
### Reproduction steps

```bash
cmake -Bbuild -H. -GNinja -DCMAKE_INSTALL_PREFIX=""
ninja -Cbuild
export DESTDIR=/someexistinglocation
ninja -Cbuild install
```

### Expected behavior

libgit2 is installed in `/someexistinglocation`.

### Actual behavior

The following error is produced:

```
CMake Error at src/cmake_install.cmake:49 (file):
file called with network path DESTINATION.  This does not make sense when
using DESTDIR.  Specify local absolute path or remove DESTDIR environment
variable.

DESTINATION=

//lib/pkgconfig
Call Stack (most recent call first):
cmake_install.cmake:42 (include)
```

Note that if DESTDIR variable is not set, the installation succeeds with the following output:

```
ninja: Entering directory `build'
[0/1] Install the project...
-- Install configuration: "Debug"
-- Installing: //lib/pkgconfig/libgit2.pc
-- Installing: /lib/libgit2.so.0.99.0
-- Installing: /lib/libgit2.so.0.99
-- Installing: /lib/libgit2.so
-- Installing: /include/git2
-- Installing: /include/git2/refs.h
…
```

The double `/` in `//lib/pkgconfig/libgit2.pc` might be exactly what is causing CMake to think that it's an URL.

### Version of libgit2 (release number or SHA1)

0.99.0

### Operating system(s) tested

Alpine Linux 3.10

**Comment by pks-t** [comment]
Thanks for your report! This should be fixed via #5412 already, so could you please let us know whether your issue is fixed already on current master?

**Comment by tpimh** [comment]
Thank you! The patch fixes the problem.
